generators/app/index.js

```diff
@@ -273,6 +273,20 @@ module.exports = JhipsterGenerator.extend({
                 local: require.resolve('../client')
             });
 
+        },
+
+        composeLanguages: function () {
```

Comment:
It generates the server an client side code and the language generator is checking the skip client and skip server commonOptions